### DISCUSSION TITLE
The Exporter & FS22

### DISCUSSION THREAD

**Discussion by StjerneIdioten** [StjerneIdioten/I3D-Blender-Addon#142] [Announcements]
Hi there,

FS22 is right around the corner and several of you might want to convert some of your FS19 mods to the new game. While I am still working on a new exporter, it won't be anywhere near ready for the FS22 launch, thus my plan is to update the existing one with any changes needed for it to be able to export for FS22 in the same capacity as it did for FS19. This means no new features, only updating any existing ones that needs tweaking for FS22.

There is also some rumours going around about Giants _finally_ providing their own exporter. The [Collectors Edition](https://www.farming-simulator.com/newsArticle.php?lang=en&country=us&news_id=235) refers to include some modding tutorials for Blender and I am pretty sure they aren't using this exporter 🤣 While I don't necessarily have high hopes for it, they might surprise 😉 None the less I will still work on my own exporter as they are surely not going to implement even half of the feature request I have laying around 😄

**Comment by JeffStanyer** [comment]
Regardless of what Giants offer, I think you have a very strong following and a lot of gratitude for the work you have done to date.
